syft/rekor/queryRekor/verify.go

```diff
@@ -0,0 +1,136 @@
+package queryRekor
+
+import (
+	"context"
+	"crypto/sha256"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/anchore/syft/internal/log"
+	"github.com/sigstore/cosign/cmd/cosign/cli/fulcio/fulcioroots"
+	"github.com/sigstore/cosign/pkg/cosign"
+	"github.com/sigstore/rekor/pkg/generated/client"
+	"github.com/sigstore/rekor/pkg/generated/models"
+)
+
+// verify the inclusion proof
+func verifyLogEntry(ctx context.Context, rekorClient *client.Rekor, entry *models.LogEntryAnon) error {
+	err := cosign.VerifyTLogEntry(ctx, rekorClient, entry)
+	if err != nil {
+		log.Debug("Error verifying Rekor entry")
+		return err
+	}
+	return nil
+}
+
+// verify the certificate
+func verifyCert(rekorClient *client.Rekor, cert *x509.Certificate) error {
+
+	certCheckOpts.RekorClient = rekorClient
+	// this code is now in sigstore, not cosign. Will break when cosign dependency updates
```

Comment:
hmm what do you mean by this?

Reply:
This package was moved from sigstore/cosign to sigstore/sigstore. I updated the Syft dependencies to reflect the change.